src/base/BaseScene.cc

```diff
@@ -1350,6 +1381,15 @@ void BaseScene::CreateMaterials()
   material->SetReceiveShadows(false);
   material->SetLightingEnabled(false);
 
+  material = this->CreateMaterial("Default/TransPurple");
+  material->SetAmbient(1.0, 0.0, 1.0);
+  material->SetDiffuse(1.0, 0.0, 1.0);
+  material->SetEmissive(1.0, 0.0, 1.0);
+  material->SetTransparency(0.5);
+  material->SetCastShadows(false);
+  material->SetReceiveShadows(false);
+  material->SetLightingEnabled(false);
```

Comment:
looks like the semi-transparent purple box is occluding the green lines in the ogre2 visualization. I think you can disable depth write to make the transparency effect more like ogre 1.x (which has depth write disabled by default when objects are semi-transparent).

```
material->SetDepthWriteEnabled(false);
```

Reply:
Fixed in [`42a6a414`](https://github.com/ignitionrobotics/ign-rendering/pull/326/commits/42a6a4143d1a6729a3d1b8a2b54e3d23d8970669)